.travis.yml

```diff
@@ -22,6 +22,8 @@ matrix:
       env: WP_VERSION=3.7.11
     - php: 5.3
       env: WP_VERSION=3.7.11 DEPLOY_BRANCH=master
+    - php: 7.1
+      env: WP_VERSION=latest SAVE_COMMENTS_DISABLED=1
```

Comment:
No, that was running the whole thing with save_comments disabled, sorry missed that. Do you mean just run it on a subset of the features? (The tag `@require-opcache-save-comments` is only to avoid a test.)